perfkitbenchmarker/scripts/database_scripts/launch_driver.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script drives launch_my_sql service which is a wrapper class for
+# mysql_service_benchmark. 
+# Driver provisions a new instance approximately every three days.
+# The run phase is completed several times daily with results uploaded to
+# a designated Google Cloud Storage Bucket.
+
+# If desired, per second data graphs can be created and uploaded to a designated
+# Google Cloud Storage Bucket.
+
+# The values below are provisioned for realistic benchmarking.
+
+# This script is designed for benchmarking purposes. Therefore, a value for
+# a Google Cloud Storage Bucket must be passed in when called.
+
+# Example call:
+#    ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh bucket_name
+
+# To utilize per second data graphs, include a storage bucket as a second input.
+# Example:
+#     ./perfkitbenchmarker/scripts/database_scripts/launch_driver.sh log_bucket_name graph_bucket_name
+
+
+if [ -z $1 ]; then
+  echo "Must pass in a cloud storage bucket."
+  exit 1;
+fi
+
+while true
+  do
+  # provision, prepare phase of mysql_service
+  run_uri=$(python perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py --sysbench_run_seconds="1200" --run_stage=provision,prepare --mysql_svc_db_instance_cores="16" --mysql_svc_oltp_table_size="12000000" --mysql_svc_oltp_tables_count="100" --mysql_instance_storage_size=1000 --additional_flags='"'"--cloud_storage_bucket=${1}"'"')
```

Comment:
for all the constants hard-coded in the bash commands below, please define bash constants at the beginning of this file and then refer to these constants in the commands. That way it becomes easy for the user to manage these constants.